tokio-console/src/config.rs

```diff
@@ -62,11 +63,29 @@ pub struct Config {
     /// * `lost-waker` -- Warns when a task is dropped without being woken.
     ///
     /// * `never-yielded` -- Warns when a task has never yielded.
-    ///
     #[clap(long = "warn", short = 'W', value_delimiter = ',', num_args = 1..)]
     #[clap(default_values_t = KnownWarnings::default_enabled_warnings())]
     pub(crate) warnings: Vec<KnownWarnings>,
 
+    /// Allow lint warnings.
+    ///
+    /// This is a comma-separated list of warnings to allow.
+    ///
+    /// Each warning is specified by its name, which is one of:
+    ///
+    /// * `self-wakes` -- Warns when a task wakes itself more than a certain percentage of its total wakeups.
+    ///                  Default percentage is 50%.
+    ///
+    /// * `lost-waker` -- Warns when a task is dropped without being woken.
+    ///
+    /// * `never-yielded` -- Warns when a task has never yielded.
+    ///
+    /// If this is set to `all`, all warnings are allowed.
+    ///
+    /// [possible values: all, self-wakes, lost-waker, never-yielded]
```

Comment:
I hope to set this automatically. But because we are using `FromStr` to parse value, we can not get it.

Reply:
Can we switch to using `clap`'s `ValueEnum` derive, so that we can have it automatically generate this?

Reply:
ValueEnum only supports unit variants. `error: `#[derive(ValueEnum)]` only supports unit variants. Non-unit variants must be skipped`

Reply:
Ah, that makes sense. Never mind. It might be nice to add a comment on the enum stating that we can't derive `ValueEnum` for it?

Reply:
Good suggestion.Thanks! Added.